<launch>

  <node pkg="etsi_visualization" type="ETSIViz" name="ETSIViz" output="screen">
    <param name="SPAT_Topic_Name" value="/TopicSPATs" /> <!-- fill in here your choosen topic name -->
    <param name="MAP_Topic_Name" value="/TopicMAPs" /> <!-- fill in here your choosen topic name -->
    <param name="CAM_Topic_Name" value="/CAM" />
  </node>

</launch>
